specification/cognitiveservices/data-plane/LUIS/Authoring/readme.go.md

```diff
@@ -12,9 +12,11 @@ go:
 ### Go multi-api
 
 ``` yaml $(go) && $(multiapi)
+
```

Comment:
This extra space needs to be removed.  Also, please remove `azure-sdk-for-go` from the `swagger-to-sdk` section in `readme.md`.  That's for track 1 code generation and we won't be generating this SDK for track 1.